firmware/src/hal/sgx/src/trusted/secret_store.c

```diff
@@ -48,18 +49,64 @@
 
 #define MAX_BLOB_SIZE (1024 * 1024)
 
+#define SECRET_PAYLOAD_SIZE(secret_size) (secret_size - sizeof(secret_header_t))
+#define SEST_HASH_SIZE (32)
```

Comment:
Can we use `HASH_LENGTH` from `hal/constants.h` instead so we avoid duplicating constants?